MongoDB/containers/0.1.0/mongodb-config/connect.sh

```diff
@@ -27,7 +30,7 @@ function find_master {
 }
 # Script starts here
 # wait for mongo to start
-while [ `$DIG A $MONGO_SERVICE_NAME +short | wc -l` -lt 3 ]; do
+while [ `$GIDDYUP ip stringify --delimiter " " | wc -w` -lt 3 ]; do
```

Comment:
I think you want `giddyup service wait scale`... check the docs.. but essentially it gates until set scale and the number of containers that exist are equal.